server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityAppContainer.js

```diff
@@ -246,15 +250,15 @@ class ProjectActivityAppContainer extends React.PureComponent {
     });
   }
 
-  updateGraphData = (graph: string, customMetrics: Array<string>) => {
+  updateGraphData = (graph /*: string */, customMetrics /*: Array<string> */) => {
     const graphMetrics = getHistoryMetrics(graph, customMetrics);
     this.setState({ graphLoading: true });
-    this.fetchMeasuresHistory(graphMetrics).then((measuresHistory: Array<MeasureHistory>) =>
+    this.fetchMeasuresHistory(graphMetrics).then((measuresHistory /*: Array<MeasureHistory> */) =>
       this.setState({ graphLoading: false, measuresHistory })
     );
   };
 
-  updateQuery = (newQuery: Query) => {
+  updateQuery = (newQuery /*: Query */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)